deploy.sh

```diff
@@ -315,6 +328,50 @@ add_hosts_to_hbi() {
   echo "Done. [AFTER_COUNT: ${AFTER_COUNT}, TARGET_COUNT: ${TARGET_COUNT}]"
 }
 
+create_hbi_connectors() {
+  echo "Creating HBI debezium connectors..."
+  NAMESPACE=env-$(oc project -q)
+  oc process -f ./deploy/debezium-connector-hosts.yml -p KAFKA_CONNECT_INSTANCE="$NAMESPACE" | oc apply -f -
```

Comment:
As it stands right now, deploying KIC will also deploy the kessel-kafka-connect cluster until future changes. 
Since updating Inventory API to do it instead might slow down the script too much, we might just want to leave it in KIC so that its optional and doesnt impact others just yet. 

With that in mind, it would be good to set `KAFKA_CONNECT_INSTANCE` to `kessel-kafka-connect` so that it uses our connect instead of clowders so we can test it more before stage/prod